dashboard/db/migrate/20160809170542_add_view_answers_unlocked_at_to_user_levels.rb

```diff
@@ -0,0 +1,6 @@
+class AddViewAnswersUnlockedAtToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :view_answers, :boolean
+    add_column :user_levels, :unlocked_at, :datetime
```

Comment:
A heads up, last time we migrated the `UserLevel` table we had to pause writes (even then it took a few hours).  +cc @breville who ran the last migration to add `:submitted` and @ashercodeorg.

Are there other changes we should consider making at the same time?  E.g. we've talked about whether the `:submitted` boolean should maybe be an enum.